schema/json/schema-latest.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "anchore.io/schema/syft/json/16.0.18/document",
+  "$id": "anchore.io/schema/syft/json/16.0.19/document",
```

Comment:
Can you help me understand why there's a schema change? As I understand it we're not affecting Syft JSON output at all.